.github/workflows/test.yml

```
@@ -0,0 +1,36 @@
+        override: true
+    
+    - name: Install dependencies
+      run: |
+        pip install uv
+        uv sync
+    
+    - name: Check for mocks
+      run: python scripts/validate-no-mocks.py
+    
+    - name: Run tests
+      run: pytest tests/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)